docs/swagger/spec.yml

```diff
@@ -321,6 +476,89 @@ definitions:
         type: string
         description: URL safe version of name
         readOnly: true
+  LayerSource:
+    allOf:
+      - $ref: '#/definitions/BaseModel'
+      - type: object
+        properties:
+            name:
+              type: string
+              description: Display name for the layer
+            sourceURI:
```

Comment:
I'm not sure there will be a sourceURI for layers, there will be for Layer Images though.